phpunit.xml.dist

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/4.4/phpunit.xsd"
+         colors="true"
+         verbose="true"
+         bootstrap="vendor/autoload.php">
+    <testsuites>
+        <testsuite name="reCAPTCHA Test Suite">
```

Comment:
This looks like the wrong name. What about "php-photoslibrary Test Suite"?

Reply:
copypasta, my bad. will change later :)

Reply:
resolved in 5b5e9c6